config/discogs-genres.yml

```diff
@@ -0,0 +1,627 @@
+Acid:
+  label: Psychedelic rock music

```

Comment:
This file should be in lib/generators/qa/discogs/templates/config only.  There should be instructions on how to run the generator that will put this in an app.

Reply:
done